src/api.jl

```diff
@@ -48,7 +48,11 @@ function savefig2(p::Plot, fn::AbstractString; dpi::Real=96)
 end
 
 # an alternative way to save plots -- no shelling out, but output less pretty
-function savefig(p::Plot, fn::AbstractString,
+# js can be one of
```

Comment:
This comment section would be great to include as a docstring instead of just comments